web/job_specs_controller_test.go

```diff
@@ -27,17 +52,25 @@ func TestJobSpecsController_Index(t *testing.T) {
 	app.Store.SaveJob(&j2)
 
 	resp := cltest.BasicAuthGet(app.Server.URL + "/v2/specs")
-	assert.Equal(t, 200, resp.StatusCode, "Response should be successful")
+	return &j1, resp
+}
 
-	var jobs []models.JobSpec
-	json.Unmarshal(cltest.ParseResponseBody(resp), &jobs)
-	assert.Equal(t, j1.Initiators[0].Schedule, jobs[0].Initiators[0].Schedule, "should have the same schedule")
-	assert.Equal(t, models.InitiatorWeb, jobs[1].Initiators[0].Type, "should have the same type")
-	assert.NotEqual(t, true, jobs[1].Initiators[0].Ran, "should ignore fields for other initiators")
+func BenchmarkJobSpecsController_Create(b *testing.B) {
+	app, cleanup := cltest.NewApplication()
+	defer cleanup()
+
+	b.ResetTimer()
+	for n := 0; n < b.N; n++ {
+		j := cltest.FixtureCreateJobViaWeb(b, app, "../internal/fixtures/web/hello_world_job.json")
+		var initr models.Initiator
+		app.Store.One("JobID", j.ID, &initr)
```

Comment:
I think it'd be a good idea to remove the check in the database. Maybe check for a 200 status code, but testing that the API call actually works is handled in another test.